src/Raven.Server/Documents/Patch/ScriptRunner.cs

```diff
@@ -293,39 +292,65 @@ private void FillDoubleArrayFromJsArray(double[] array, ArrayInstance jsArray, s
                     ++i;
                 }
             }
-            
-            private JsValue AppendTimeSeries(JsValue self, JsValue[] args)
+
+            private JsValue TimeSeries(JsValue self, JsValue[] args)
+            {
+                AssertValidDatabaseContext("timeseries(doc, name)");
```

Comment:
Do we want to have `timeseries(name)` also and Id would be taken (IIRC) from `SingleRun.OriginalDocumentId` ?

Reply:
You might be called on another document, though. Maybe loaded.

Reply:
This is why I thought to have it as an addition to the current one